bottom/client.py

```diff
@@ -1,14 +1,20 @@
 import asyncio
 import collections
 import functools
+from typing import Any, Callable, DefaultDict, List, Optional  # noqa
 from bottom.protocol import Protocol
 from bottom.pack import pack_command
 
 
 class Client:
-    protocol = None
+    protocol = None  # type: Optional[Protocol]
 
-    def __init__(self, host, port, *, encoding="UTF-8", ssl=True, loop=None):
+    _handlers = None  # type: DefaultDict[str, List[Callable]]
+    _events = None  # type: DefaultDict[str, asyncio.Event]
+
+    def __init__(self, host: str, port: int, *,
+                 encoding: str = "UTF-8", ssl: bool = True,
```

Comment:
It doesn't strictly recommend it, but examples use this style. Mypy codebase uses this as well. I can change it, but I think the community will stick to version with spaces (given annotation is present).